src/cli/commands/force/lightning/lwc/test/ui/mobile/configure.ts

```diff
@@ -364,14 +367,20 @@ export class Configure extends SfdxCommand implements HasRequirements {
         }
 
         // Convert to JSON, then "sanitize" and convert from JSON to a valid JS.
+        const chromeDriverAutoDownloadComment =
+            '// fetch the appropriate version of chrome driver as needed';
         const regex = /"\$\{\w+\}"/g;
-        const configJSON = JSON.stringify(config, undefined, 2).replace(
+        let configJSON = JSON.stringify(config, undefined, 2).replace(
             regex,
             (match) => {
                 const key = match.slice(1, -1);
                 return placeholder_values.get(key) ?? match;
             }
         );
+        configJSON = configJSON.replace(
```

Comment:
Jumping through these hoops (plus the hoops in lines 301-317 & 375-377) b/c we're defining `config` in line 319 as an object instead of a string template.

@khawkins The more I touch this code them more I'm thinking that perhaps we should have just used a string template and avoided all of these special cases.